app/models/custom_font_and_color_theme_updater.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+class CustomFontAndColorThemeUpdater
+  VISUAL_DESIGN_FONT = %w[primary_navigation_font_family header_links_font_family footer_and_results_font_family identifier_font_family].freeze
```

Comment:
This appears to be the same as `Affiliate::FONT_FIELDS`. We shouldn't duplicate any of these attributes in these constants.